server/serverAPI/controller/InventoryController.ts

```diff
@@ -61,11 +114,28 @@ export default class InventoryController extends BaseUserController {
     getAll = async (req: Request, res: Response) => {
         let parameters = new Map<string, any>([["username", req.serverUser.username]]);
 
+        let isReverse = req.query.isReverse === 'true' ? true : false;
```

Comment:
I don't understand what you mean by that... give me an example of how it will look

Reply:
I'm saying we should require that req.query.reverse will be a boolean, not a string, so you don't need this check

Reply:
we can't because u can't pass type into the query... query is a part of URL, URL doesn't have types. Everything that is coming from req.query will be of type string no matter what you are going to do because that's how browsers send it.